src/lib/error/ErrorReport.ts

```diff
@@ -39,15 +48,27 @@ interface LogToUserOptions {
 }
 
 export class ErrorReport extends ErrorReportBase {
-  public static create(args: ErrorReportCreateArgs) {
-    return new ErrorReport(createErrorReportBaseArgs(args))
+  public static checkIfShouldRemoteReport(err: Error | any) {
```

Comment:
why you chose 
```
if (err instanceof CommandError) {
  return false
}

return true
```
instead of 
`
return (err instanceof CommandError == false)
`

Reply:
For now there's only `CommandError` type that shouldn't be reported, but we may want to add more types later - I think the more verbose way to check the instances is more understandable in this case